frontend/ui/screensaver.lua

```diff
@@ -192,8 +193,10 @@ function Screensaver:expandSpecial(message, fallback)
         local powerd = Device:getPowerDevice()
         if Device:hasAuxBattery() and powerd:isAuxBatteryConnected() then
             batt_lvl = powerd:getCapacity() + powerd:getAuxCapacity()
+            batt_symbol = powerd:getBatterySymbol(powerd:isAuxCharged(), powerd:isAuxCharging(), aux_batt_lvl)
```

Comment:
I was a bit too enthusiastic. You should also add a local batt_symbol = something up above where the same is done for batt_lvl.

Reply:
I corrected the problem and the linter no longer throws errors. I also made a change so the icon acts the same as the footer as explained in the comment below. Though @NiLuJe said he had tentative concerns over the feature and I haven't heard back from him.